monitor.py

```diff
@@ -111,6 +114,9 @@ def __init__(self) -> None:
         self.systempopulation: Optional[int] = None
         self.started: Optional[int] = None  # Timestamp of the LoadGame event
 
+        self._navroute_retries = 0
+        self._navroute_orig_time: Optional[float] = None
```

Comment:
I was going to make another "add this comment" comment, but instead I think the variable name should be changed to `self._navroute_journal_timestamp`.